packages/dds/cell/src/cell.ts

```diff
@@ -124,6 +132,8 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>>
      */
     private messageIdObserved: number = -1;
 
+    private readonly pendingMessageIds: number[] = [];
```

Comment:
Entries should be removed when the corresponding op is ack'ed. Otherwise this will grow forever and contain things that should not ever be rolled back (only edits that are still completely local can be rolled back).